packages/metrics/src/node/extensions-metrics-contribution.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+import * as yargs from 'yargs';
+import { injectable } from 'inversify';
+import { MetricsContribution } from './';
+import { ApplicationPackageManager } from '@theia/application-package';
+
+const appProjectPath = 'app-project-path';
+
+@injectable()
+export class ExtensionMetricsContribution implements MetricsContribution {
+    private metrics: string = "";
+    readonly applicationPackageManager: ApplicationPackageManager;
+
+    getMetrics(): string {
+        return this.metrics;
+    }
+
+    startCollecting(): void {
+        const projectPath = yargs.parse(process.argv)[appProjectPath];
```

Comment:
When I start Theia with ```yarn run start``` from ```examples/browser```, I noticed that ```projectPath``` ends-up being undefined. It works however if I add this switch ```--app-project-path=$CWD```

Reply:
I just noticed that the extension-manager has a default for `process.cwd` if no arguments are supplied [here](https://github.com/theia-ide/theia/blob/master/packages/extension-manager/src/node/application-project-cli.ts#L32). I'll do something similar.